integration/travis/scheduler_config.edn

```diff
@@ -16,6 +16,10 @@
          :failover-timeout-ms nil
          :leader-path "/cook-scheduler"
          :role "cook"}
+ :executor {:command "/home/travis/build/twosigma/Cook/travis/cook-executor"
```

Comment:
Is it possible to have travis use the uri instead of have it deployed to the machines? This will ensure that code path always works.